lib/awful/tooltip.lua

```diff
@@ -546,6 +551,22 @@ function tooltip:set_margins_topbottom(val)
     self:set_margin_topbottom(val)
 end
 
+--- Set the margins between the tooltip and its parent
+--
+-- @DOC_awful_tooltip_gaps_EXAMPLE@
+--
+-- @property tooltip_gaps
+-- @tparam number|table New margins value
```

Comment:
Add a `.` at the end of the line. This is annoying, but ldoc is often annoying.

Reply:
Should I add `.` to all the lines without it?

Reply:
Just the lines with a description and the first line with the `---`. For the later, `ldoc` uses that for the summary in the table of content. It will put everything until the first `.`, even if it is over multiple lines. For the `@tsomething`, not having the `.` usually works, but sometime triggers weird bugs. It is better to just have it everywhere.